src/rank_llm/retrieve/retriever.py

```diff
@@ -17,6 +17,7 @@
 class RetrievalMode(Enum):
     DATASET = "dataset"
     CUSTOM = "custom"

```

Comment:
what's the exact difference between CUSTOM and LOAD_FROM_FILE? Maybe CACHED_FILE is a name choice here?